.ci/jobs/fleet-server.yml

```
@@ -13,7 +13,7 @@
         discover-pr-forks-trust: permission
         discover-pr-origin: merge-current
         discover-tags: true
-        head-filter-regex: '^(?!update-.*-version).*$'
+        head-filter-regex: '(master|7\.x|7\.1\d|8\.\d+|PR-.*|v\d+\.\d+\.\d+|(?!update-.*-version).*$)'
```

Using adapted regex from Beats repo

---

`(?!update-.*-version)` is not needed then

```suggestion
        head-filter-regex: '(master|7\.x|7\.1\d|8\.\d+|PR-.*|v\d+\.\d+\.\d+)'
```

---

OK, I kept it in the case it was needed, but you are right: it's excluding that pattern. 👍